### ISSUE TITLE
The bot doesn't have the permission to open PR anymore

### ISSUE THREAD

**Issue by saschanaz** [microsoft/TypeScript-DOM-lib-generator#1865]
https://github.com/microsoft/TypeScript-DOM-lib-generator/actions/runs/12211296199/job/34068390025

Did the PAT change mentioned in #1859 break things? @jakebailey

**Comment by jakebailey** [comment]
Yeah, annoying. Can we just use dependabot instead, or do these PRa typically do something other than update package.json/locks?

**Comment by saschanaz** [comment]
The bot is there to regenarate the libs based on the new updates and file PR only if there's any diffs made in the libs. I don't think dependabot can do that.

**Comment by saschanaz** [comment]
Can we reconsider the change? The change stopped the core maintenance logic.

**Comment by jakebailey** [comment]
I'll try and fix it; I've just been out for the past few weeks.

**Comment by jakebailey** [comment]
Running again: https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1870
